arcgis-ios-sdk-samples/Utility Network/Connected Trace/ConnectedTraceViewController.swift

```diff
@@ -235,18 +235,18 @@ class ConnectedTraceViewController: UIViewController, AGSGeoViewTouchDelegate {
                     guard let features = features else { return }
                     
                     layer.select(features)
-                })
+                }
             }
             
-            selectionGroup.notify(queue: .main, execute: { [weak self] in
+            selectionGroup.notify(queue: .main) { [weak self] in
                 self?.setStatus(message: "Trace completed.")
                 SVProgressHUD.dismiss()
-            })
+            }
         }
     }
     
     func clearSelection() {
-        map.operationalLayers.lazy.compactMap({ $0 as? AGSFeatureLayer }).forEach({ $0.clearSelection() })
+        map.operationalLayers.lazy.compactMap { $0 as? AGSFeatureLayer }.forEach { $0.clearSelection() }
```

Comment:
When performing chained operations on a collection, I generally like to put each operation on a separate line:

```suggestion
        map.operationalLayers.lazy
            .compactMap { $0 as? AGSFeatureLayer }
            .forEach { $0.clearSelection() }
```

I think it's because a dot hanging off of a curly brace looks a little to me. But that's just me.

Reply:
Agreed. I'll tidy this up as well.